.pre-commit-config.yaml

```diff
@@ -1,5 +1,11 @@
 repos:
 
+- repo: https://github.com/psf/black
+  rev: 20.8b1
+  hooks:
+    - id: black
+      language_version: python3.7
```

Comment:
do we need language_version?

Reply:
This is how they have it in the [black documentation](https://github.com/psf/black#version-control-integration) so I stuck with it.